docs/docs/20-quickstart.md

```diff
@@ -508,52 +510,28 @@ the previous section.
     EOF
     ```
     
-  :::info
-  Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
-  :::
-
-1. Use the CLI to view our `Warehouse` resource:
-
-    ```shell
-    kargo get warehouses --project kargo-demo
-    ```
+      :::info
+      Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
+      :::
 
-    Sample output:
+1. After applying the above configuration, you can view the pipeline you just created using the [Kargo Dashboard](https://localhost:31444/).
```

Comment:
Since this is an ordered list of steps that the reader is expected to follow, I think it's well-understood that this step comes after the previous one and we don't need to waste words on that.

```suggestion
1. View the pipeline you just created using the [Kargo Dashboard](https://localhost:31444/):
```